src/modules/ContactsPage/ContactsPage.module.css

```diff
@@ -0,0 +1,91 @@
+.promo {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    /* padding-left: 74px; */
+    padding-left: 3.85vw;
+    /* padding-bottom: 36px; */
+    padding-bottom: 3.96vh;
+    /* height: 718px; */
+    height: 78.90vh;
+    background: url(../../resources/img/ContactsPage.webp) center center/cover no-repeat;
+    z-index: -1;
+    margin-bottom: 3.85vh;
+}
+
+.promoDescr {
+    /* font-size: 48px; */
+    font-size: 5.27vh;
+    font-weight: 400;
+    /* line-height: 58px; */
+    line-height: 6.37vh;
+    color: rgba(255, 255, 255, 1);
+    z-index: 1;
+}
+
+.promoDescr span {
+    font-weight: 600
+}
+
+.promoRectangle {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    background-color: rgba(0, 0, 0, .4);
+    z-index: 0;
+}
+
+.promoContacts {
+    /* padding: 0 117px 0 254px; */
+    padding: 0 6.09vw 0 13.23vw;
+}
+
+.promoTitle {
+    /* font-size: 48px; */
+    font-size: 5.27vh;
+    font-weight: 600;
+    /* line-height: 58px; */
+    line-height: 6.37vh;
+    color: rgba(75, 125, 62, 1);
+    /* margin-bottom: 33px */
+    margin-bottom: 3.63vh
+}
+
+.promoWrapper {
+    display: grid;
+    /* grid-template-columns: 657px 727px; */
+    grid-template-columns: 34.22vw 37.86vw;
+    /* column-gap: 162px; */
+    column-gap: 8.44vw;
+    /* margin-bottom: 26px; */
+    margin-bottom: 2.86vh;
+}
+
+.promoPositions {
+    /* font-size: 24px; */
+    font-size: 2.64vh;
+    font-weight: 400;
+    /* line-height: 30px; */
+    line-height: 3.30vh;
+}
+
+.promoPositions span {
+    font-weight: 700;
+}
+
+.promoEmail {
+    /* font-size: 48px; */
+    font-size: 5.27vh;
+    font-weight: 700;
+    /* line-height: 58px; */
+    line-height: 6.37vh;
+    text-align: center;
+}
+
+.promoEmail a {
+    text-decoration: none;
+    color: rgba(75, 125, 62, 1);
+}
```

Comment:
remove comments
and please use relative values not for all sizes, but only where it need
![image](https://github.com/moskalevNick/okb-client/assets/76160475/89703b11-7511-4aac-a05c-78daeeac05f9)